src/gui/accountsettings.cpp

```diff
@@ -202,6 +202,7 @@ void AccountSettings::slotOpenAccountWizard()
 void AccountSettings::slotToggleSignInState()
 {
     if (_accountState->isSignedOut()) {
+        _accountState->account()->resetRejectedCertificates();
```

Comment:
I made it like that on purpose because i was thinking about the scenrario in which the user has two accounts: one with a valid account, and one for which the certificate is invalid for an unknown reason (attacker eavesdropping?) for several days. After having rejected the certificate, we want to keep this account offline.  But when the user wants to go online with the other account by clicking on the system tray, we don't want to show the dialog again for the broken server.  Or do we?